packages/widget/src/themes/jumper.ts

```diff
@@ -0,0 +1,37 @@
+import type { WidgetTheme } from '../types/widget.js';
+
+export const jumperTheme: WidgetTheme = {
```

Comment:
This theme doesn't seem to do a lot - is it still needed?

Reply:
I'll update it later this week 🙂